src/scripts/main.js

```diff
@@ -354,7 +354,26 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+function createTableRow(tableId) {
+  people.age = people.born - people.died;
+  people.century = Math.ceil(people.died / 100);
 
-// write your code here
+  const table = document.querySelector('dashboard');
+
+  const properties = ['name', 'age', 'city', 'job', 'phone', 'email'];
+
+  people.forEach((person) => {
+    const row = document.createElement('tr');
+
+    properties.forEach((prop) => {
+      const cell = document.createElement('td');
+
+      cell.textContent = person[prop] || '';
+      row.appendChild(cell);
+    });
+
+    table.appendChild(row);
+  });
+}
+
+createTableRow();
```

Comment:
The 'createTableRow' function is called without any arguments, but it expects a 'tableId' parameter. Ensure you pass the correct table ID when calling this function.